bot.py

```diff
@@ -1,117 +1,41 @@
-    if num == 1:
-        body = "To be fair, you have to have a very high IQ to understand Arsenal football. The tactics are extremely subtle, and without a solid grasp of the 3-4-2-1 formation most of the crosses will go over a typical striker’s head. There’s also Wenger’s nihilistic outlook, which is deftly woven into his lineup- his personal philosophy draws heavily from Herbert Chapman literature, for instance. The fans understand this stuff; they have the intellectual capacity to truly appreciate the depths of these losses, to realise that they’re not just funny- they say something deep about LIFE. As a consequence people who dislike the Arsenal truly ARE idiots- of course they wouldn’t appreciate, for instance, the humour in the Gunners’ existential catchphrase “Wenger Out!” which itself is a cryptic reference to the Invincibles Season. I’m smirking right now just imagining one of those addlepated simpletons scratching their heads in confusion as Mesut Ozil’s genius wit unfolds itself on their television screens. What fools.. how I pity them.  😂\n And yes, by the way, i DO have an Arsenal tattoo. And no, you cannot see it. It’s for the ladies’ eyes only- and even then they have to demonstrate that they’re within 5 IQ points of my own (preferably lower) beforehand. Nothin personnel kid  😎"
-    return body
+prefix = '!'
+bot = commands.Bot(
+    command_prefix='!',
+    case_insensitive=True
+)
+
+
+def load_cogs():
+    # Set cogs that require loading in a specific order
+    # They will be loaded in the order of the list, followed by all
+    # other cogs in the /cogs folder
+    cogs = [
+    ]
+
+    if os.path.exists("./cogs"):
+        for file in os.listdir("./cogs"):
+            if file.endswith(".py"):
+                cog_name = file[:-3]
```

Comment:
Is this the recommended way to check if there are cogs files? Seems quite odd

Reply:
This is in no way a recommended way of doing things but personally, I've found this much easier to work with. Instead of having to come and update some list of cogs when you make a new one or change it's name, it all just works.

Only negative imo is the setup function at the bottom of each cog file but that's not too bad tbh

Reply:
Yeah, the setup bit is fine. In the future, we could look into automating that part too, but for now we should be good